### PR TITLE
feat(cli): delete management token on logout

### PR DESCRIPTION
**Pull request by jasonkuhrt**
This PR makes the CLI invalidate a management token when the user logs out.

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -126,7 +128,8 @@
     "prepublishOnly": "pnpm run build"
   },
   "dependencies": {
-    "@prisma/engines": "workspace:*"
+    "@prisma/engines": "workspace:*",
+    "jose": "^5.2.2"
```
Comment (janpio) [inline packages/cli/package.json:132]: Is this in the correct place? All the other things seem to be in `devDep`.

Comment (jasonkuhrt) [reply packages/cli/package.json:132]: Its not that complex, let's inline it.

https://github.com/panva/jose/blob/05c4351be3a356da2a0e882fbbd8006f2725ec7b/src/util/decode_jwt.ts#L22

**Review by tatianainama — APPROVED** [review]

**Comment by jasonkuhrt** [comment]
Refactored, looking good:

CleanShot 2024-02-19 at 12 35 37@2x

File: packages/cli/package.json (hunk 1 above)
Comment (millsp) [reply packages/cli/package.json]: If you move it to `devDepenbdencies` we will bundle it automatically. And if it has an ESM export then we would also tree-shake the unused imports. If it's small enough you don't need to worry about inlining.

**Review by millsp — APPROVED** [review]

**Pull request merged**